RELEASE/scripts/sl_ascend.ash

```diff
@@ -92,10 +92,12 @@ void initializeSettings()
 	if(my_pool.find() && (my_turncount() == 0))
 	{
 		int curSkill = to_int(my_pool.group(1));
```

Comment:
Hmm, shouldn't this also account for pool skill from equipment and drunkenness? The value in the quest log definitely changes when I equip a pool cue.

Reply:
Well that explains why sl_ascend was clobbering people's pool skill. Ugh. I guess we can use numeric_modifier on all current equipment/buffs to get our base pool skill?

```
> ash (numeric_modifier($item[pool cue], "pool skill"))

Returned: 3.0
```

Reply:
We already do all the equipment calcs when we're deciding on the billiards room; this was just being set upon initialization. IIRC drunkenness doesn't update the quest log value, so this would only be an issue if someone equipped pool gear and then ran sl_ascend on turn 0, then followed the recommended instruction to manually set their SR count.